src/ext/JourneyPatternStopPointMap/JourneyPatternStopPointMap.tsx

```diff
@@ -38,27 +74,88 @@ export const JourneyPatternStopPointMap = ({
       stopPointLocationSequence: [],
       showQuaysState: {},
       hideNonSelectedQuaysState: {},
+      focusedMarker: undefined,
     },
   );
+  const mapStateRef = useRef<MapState>({
```

Comment:
I'm trying to understand the usage of ref here, but struggling a bit. I might need a deeper explanation.